### PR TITLE
MKT-934 -- Send butterbar param to eventbrite when present on /opticon

### PR DESCRIPTION
**Pull request by stewsmith**
https://optimizely.atlassian.net/browse/MKT-934

This PR is so we can see how many people are buying tickets through the butterbar in the product.

It passes the url param aff=butterbar on to eventbrite.

### CODE REVIEW

File: website-guts/assets/js/pages/opticon.js [hunk 1]
```diff
@@ -0,0 +1,3 @@
+if (window.optly.mrkt.utils.getURLParameter('aff') === 'butterbar') {
```
Comment (jgaul) [inline website-guts/assets/js/pages/opticon.js]: you could make it more flexible and just say that if any "aff" value is set, that param and its value should be written into the registration href.

Comment (stewsmith) [inline website-guts/assets/js/pages/opticon.js]: Good idea.

Comment (dtothefp) [inline website-guts/assets/js/pages/opticon.js]: @stewsmith you should probably use the deparam function for that then

File: website-guts/assets/js/pages/opticon.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
+var urlParams = window.optly.mrkt.utils.deparam(window.location.search);
+if (urlParams) {
+  var replacementHref = window.optly.mrkt.utils.param('https://opticon2015.eventbrite.com/?ref=elink', urlParams);
```
Comment (jgaul) [inline website-guts/assets/js/pages/opticon.js]: i don't know these utils, but as long as this worked when you tested it, LGTM.

Comment (stewsmith) [inline website-guts/assets/js/pages/opticon.js]: Worked when tested with 0, 1, and multiple params.

**Pull request merged**